genie-web/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
      *******************************/
 
     compile(project(":genie-core"))
+    compile project(":genie-proto")
```

Comment:
inconsistent with how above project dependency is defined

Reply:
Done